.eslintrc.json

```diff
@@ -47,7 +47,6 @@
         "space-before-function-paren": "off",
         "standard/no-callback-literal": "off",
 
-        "jsx-a11y/no-noninteractive-tabindex": "off",
```

Comment:
I totally agree with the content of this commit but without it and [this](https://github.com/cockpit-project/cockpit/pull/12854/commits/ceaaca1d139a33ede8459fb250c3e14b79267e18) included the build would break.
This especially bad in cases where you are git bisecting.
So what I would do is either merge this whole PR squashed either squash at least this commit with https://github.com/cockpit-project/cockpit/pull/12854/commits/ceaaca1d139a33ede8459fb250c3e14b79267e18

Reply:
My bad, thought that the first commit was actually enabling that rule.